# Exploring the Climate-Wine Relationship in Oregon

Do climate variables affect wine yield in Oregon, and if so, when in the growing season do they matter most?

Jennifer Arreola  
Rebekah M. Peterson  
Victoria Fox

# Introduction

Rising global temperatures have raised concern for the viability of climate-sensitive crops nationwide, and few crops illustrate that exposure as directly as wine grapes. In Oregon, wine grapes underpin an industry worth \$7.08 billion in annual economic activity, supporting nearly 40,000 jobs and \$715 million in annual tourism spending ([WineAmerica 2025](#ref-wineamerica2025)). Its flagship variety, Pinot Noir, is particularly exposed to the specific climate mechanisms driving that broader concern, a relationship we explore in detail throughout this report. More frequent heat waves and less predictable seasonal weather can shift vintage profiles and elevate the risk of disruptions like the wildfire smoke taint Oregon experienced amid the 2020 fires, which fell disproportionately on red wine varieties ([Oregon Wine Board Trade Resources 2025](#ref-oregonwine2025willamette)).

Oregon is home to 23 federally recognized American Viticultural Areas (AVAs), spanning distinct growing regions with meaningfully different climates. The Willamette Valley, which we split in our analysis into North and South Willamette Valley, sits near the 45th parallel under a cool, marine-influenced climate that gave rise to its Pinot Noir reputation in the first place. Further south and inland, the Rogue and Umpqua Valleys run warmer and drier, supporting a different varietal mix, while the Columbia River region spans Oregon’s northern border. Alongside Pinot Noir, Oregon growers plant substantial acreage of Pinot Gris, Chardonnay, Riesling, Gewürztraminer, and smaller amounts of Cabernet Sauvignon and Merlot in the state’s warmer regions([Oregon Wine Board Trade Resources 2025](#ref-oregonwine2025willamette)). For this analysis, we grouped Oregon’s AVAs into five broader reporting regions: North Willamette Valley, South Willamette Valley, Rogue Valley, Umpqua, and Columbia River at Large, matching the geographic level at which usable historical production data is actually available.

<iframe src="https://019fc4eb-ccc6-ac48-3271-c2190ba05ae5.share.connect.posit.cloud/" 
          width="100%" 
          height="750px" 
          style="border: none;"> </iframe>

Willamette Valley Pinot Noir represents a disproportionate share of that production. While our aim is to understand climate exposure across all of Oregon’s varietals, Pinot Noir’s scale means it is also the variety for which we have the deepest historical record to draw on. This data asymmetry shapes how we approach analysis and interpret results.

Despite this economic and analytical importance, no single, interpretable public data source currently houses Oregon wine data, let alone production data alongside the climate data needed to study it. We describe the origin of this gap in Background, and closing it is one of the central goals of this project’s data engineering work.

This report asks a specific version of the broader climate question above: how has Oregon’s changing climate affected wine yield across its growing regions since 1987, and does that sensitivity differ by variety and by point in the growing season? We answer it in two stages. First, a statistical panel and correlation analysis establishes the historical relationship between Oregon’s climate indicators and yield. Second, a staged machine learning model tests when in the growing season (spring, summer, or harvest) climate information actually becomes predictive of yield, and for which varieties. That second stage carries the more interesting finding: nearly all of the model’s predictive power arrives once summer heat data is available, with harvest-season features adding comparatively little on top. As we show later, that pattern lines up with a separate finding that late-season climate is a signal for wine quality rather than yield.

# Background

## Pinot Noir is a key varietal for the Oregon wine industry

Among wine producers, Pinot Noir is known for being a very difficult grape due to its thin skin, tight grape clusters, early ripening, and sensitivity to heat. For many years, Pinot Noir production was limited to regions like Burgundy, France that had a narrow band of warm, sunny days during the growing season paired with a pronounced overnight cooldown—ideal conditions for the slow, gentle ripening required for great Pinot Noir.

In the 1960s, the winemaker David Lett set out to find a region outside of Burgundy where he could successfully cultivate Pinot Noir vines. He settled in Oregon’s Willamette Valley, which sits on the same latitude as Burgundy (close to the 45th parallel) and has been assessed as sharing many of the same climate characteristics relevant to Pinot Noir viticultural site selection ([Jones and Hellman 2003](#ref-jonesHellman2003)), including the day-night temperature swing that is vital to Pinot Noir’s unique character. In 1979, Lett entered his Willamette Valley Pinot Noir in the Gault-Millau French Wine Olympiad, where he placed among the top Burgundies. This was the first real signal that Oregon could compete at the same level as the region that had produced the best Pinot Noirs for hundreds of years ([Oregon Wine Board Trade Resources 2025](#ref-oregonwine2025willamette)). The Willamette Valley became an official American Viticultural Area (AVA) in 1983 and has since become one of the most recognized Pinot Noir regions in the world, a status widely attributed to its warm-day, cool-night climate pattern ([Oregon Wine Board Trade Resources 2025](#ref-oregonwine2025willamette); [SommSelect 2026](#ref-sommselect2026)). This unique climate pattern is referred to as *diurnal temperature range*.

# The diurnal range effect

Warm days drive photosynthesis and sugar accumulation in the grapes, while cool nights slow the grape’s respiration rate, giving time for the acids, color compounds, and aromatic molecules to further develop rather than burning off overnight. Without this nightly cooldown, grapes ripen too fast and produce wines that taste “flat”—high in sugar without the balance of acidity and aromatic complexity that define great Pinot Noir ([Kliewer and Torres 1972](#ref-kliewer1972); [Seth D. Cohen et al. 2012](#ref-ajev2012dtr)).

An ideal diurnal range for Pinot Noir ripening is generally considered to be in the neighborhood of 15–20°C (~60-70°F). Controlled field and growth-chamber experiments on Pinot Noir have shown that raising the diurnal temperature range around the onset of ripening (*veraison*) alters organic acid metabolism and can inhibit color and phenolic accumulation in the grape ([<span class="nocase">Bennett et al.</span> 2025](#ref-frontiers2025bunchheating); [Sarah D. Cohen et al. 2012](#ref-cohen2012jexpbot)). While compression of the diurnal range has been shown to hasten ripening and shift how flavors develop in wine grapes broadly ([Seth D. Cohen et al. 2012](#ref-ajev2012dtr)), targeted studies on Malbec, Merlot, and Pinot Noir have found that diurnal range sensitivity is not universal across varieties and is particularly pronounced in Pinot Noir ([Rosas et al. 2022](#ref-pmc2022highTemp)).

When we consider the impact of climate on wine production, particularly in Oregon where Pinot Noir is the dominant variety, it is not just a question of whether summers are getting hotter *overall*, but rather what happens when *nights warm faster than days*. This is a documented feature of twentieth- and twenty-first-century climate shifts globally, generally attributed to changes in cloud cover trapping heat overnight ([Dahl 2022](#ref-ucs2022nights); [<span class="nocase">Liu et al.</span> 2024](#ref-liu2024dtr)). This is exactly the pattern we observe in Oregon’s climate record: a real warming signal in growing degree days and heat stress, alongside shrinking day-night temperature swings during ripening ([Dahl 2022](#ref-ucs2022nights); [<span class="nocase">Liu et al.</span> 2024](#ref-liu2024dtr)).

## Prior research on Oregon and Pinot Noir climate sensitivity

Climate research specific to Oregon wine has, to date, focused primarily on phenology and single-variety climate suitability rather than on production outcomes directly. Multi-decade climate analyses have documented measurable statewide warming, pushing the Willamette Valley from sitting at the cool edge of Pinot Noir suitability in the 1960-70s to sitting centered within that suitability range today ([Jones 2005](#ref-jones2005)). Work presented at the Oregon Wine Symposium has projected temperature-driven shifts in Oregon varietal suitability and harvest timing through 2100, pointing toward a potential reshuffling of the state’s variety composition as the climate continues to warm ([Skahill and Berenguer 2025](#ref-skahill2025)). Separately, phenological modeling published in the *American Journal of Enology and Viticulture* has modeled shifts in the timing of the Willamette Valley’s Pinot Noir growing season, reinforcing the variety’s particular sensitivity to the region’s climate ([Delelee et al. 2025](#ref-delelee2025)).

We draw our general climate-threshold framework (frost days, growing degree days, heat stress days) from a global synthesis of viticulture climate impacts ([<span class="nocase">Van Leeuwen et al.</span> 2024](#ref-vanleeuwen2024)). This project differs from that body of work in that, rather than modeling suitability or phenology on their own, we integrate 38 years of statewide production records directly with climate data to ask whether the warming trend has affected *production outcomes*, not just suitability.

## Filling an availability gap for Oregon wine data

Our motivation for this project is also driven by a genuine gap in publicly available data for Oregon wines. Due to the fact that 90% of U.S. wine grape acreage sits in California and Washington, USDA’s National Agricultural Statistics Service (NASS) discontinued collecting Oregon-specific grape data beginning in 2011 ([Wine Data Researcher 2022](#ref-nass2011gap)). While similar data continued to be collected by researchers in Oregon, this information existed largely as scanned PDF reports rather than structured data. This project’s data engineering work is built in part to close that gap by producing a structured, climate-linked Oregon wine dataset that does not currently exist from any single public source.

# Data

## Data sources

**Oregon wine production and pricing** data comes from the Oregon Vineyard and Winery Annual Reports, conducted since 1981 by various research groups and currently maintained by the University of Oregon ([Oregon Wine Board 2026](#ref-oregonwinecensus)). Reports prior to 2017 exist only as scanned & digital PDF documents and were ingested using AWS Textract; later years were ingested directly from structured spreadsheets.

**Climate data** was retrieved from the PRISM Climate Group, maintained by Oregon State University. PRISM provides daily gridded temperature and precipitation observations across the contiguous United States ([PRISM Climate Group, Oregon State University 2026](#ref-prism2026)). We retrieved daily observations from 1981-2024 across all 23 federally-recognized AVAs in Oregon. Zonal statistics were produced for each AVA by averaging the raster grid within each AVA boundary using the Python rasterstats library. AVA boundaries were defined by shapefiles obtained from the Alcohol and Tobacco Tax and Trade Bureau (TTB), which were converted to GeoJSON prior to their use in the ingestion script.

From the daily climate data, we derived a set of features chosen specifically for their relevance to viticulture, rather than working with raw daily temperature and precipitation directly.

| Feature | Definition |
|------------------------------------|------------------------------------|
| Frost risk days | Count of days below freezing, annually and during the April–May bud-break window |
| Coldest spring night | Minimum recorded temperature during spring bud-break |
| Growing degree days (GDD) | Accumulated heat above a 10°C (50°F) base, April–September (Winkler index), plus a April–October variant and a July 15–October 15 “veraison-to-harvest” window |
| Heat stress days | Count of days with a maximum temperature above 35°C (95°F) during the growing season |
| Vapor pressure deficit (VPD) | A measure of atmospheric dryness / evaporative demand, computed for summer (July–August) and ripening (August–September) windows |
| Diurnal temperature range (ripening) | Average day-night temperature swing during August–September, the period when grapes develop their sugar, color, and acidity |
| Harvest precipitation | Total precipitation in October, and in the September–October window |

## Data engineering pipeline

Climate and production data were ingested on a recurring Amazon Web Services (AWS) batch pipeline (one task per AVA), computed into region- and statewide-level summaries, and stored in a PostgreSQL database hosted on Railway. AWS was chosen specifically for the production-data side of the pipeline, as reconstructing tables from PDF reports required routing each report era through separate Textract-based parsers due to varied report formats, column layouts, and label conventions. For the climate data, daily PRISM rasters were cached prior to computing zonal statistics, which were then written to the database via an idempotent upsert. Running this pipeline on AWS Batch / Fargate Spot allowed each AVA to be processed as a parallel, independently retriable job, though Spot interruptions during the original run caused some days to be missed; these were identified and backfilled in a final pass before the analysis tables were built.

Two analysis-ready tables were constructed on top of the raw data:

| Panel | Coverage | Grain | Purpose |
|------------------|------------------|------------------|------------------|
| Regional panel | 1987-2024 | One row per region x variety x year | Yield modeling and region-level comparisons |
| Statewide panel | 1987-2024 | One row per variety x year | Long-run varietal-mix analysis where regional detail is unnecessary |

Both panels begin in 1987, rather than 1981, because of limitations in the production data. While the full PRISM climate dataset was available from 1981 for all AVAs, the underlying wine census reports for 1981-1986 do not contain the area-by-variety breakdown tables needed to reconstruct regional variety data. These reports *do* contain bearing and non-bearing acreage by county, which could in principle be aggregated to the region level for an acreage-only (not variety-level) time series; we did not pursue this, and instead clipped both panels to 1987 onward to keep the regional and statewide tables consistent.

Region-variety combinations were filtered to those with at least 20 years of historical data and zero-production / zero-harvest rows were dropped. This reduced the original ~1,300 region-variety-year observations to ~990 in the regional panel.

The analysis tables were exported to Parquet files, which are used as the final data references for this report to preserve reproducibility. Our long-term goal is to make the underlying database publicly available, but this is a continued work in progress.

## Ethical considerations

A few ethical considerations shaped how we approached this dataset:

1.  **Regional underrepresentation:** Underlying wine census data is reported by region, not by individual vineyard or winery, and is a voluntary industry survey rather than a full census. We recognize that this led to underrepresentation for some counties in some years.
2.  **Diversity and inclusion gaps among Oregon wine producers:** Because there are very few minority-owned wineries in Oregon—our research turned up only two Native-owned wineries statewide—there is a real possibility that patterns related to these ownership demographics are invisible in this dataset. The effect of climate may have a disproportionate impact on some groups over others, but data was never collected that could surface these patterns.
3.  **Impact of immigration on Oregon wine production:** Recent immigration policy has affected farmworkers and their families in ways that are impossible to quantify from production statistics alone. The truth is that Oregon wine production relies heavily on seasonal immigrant labor, and the raw production numbers do not tell the human story of these populations who are increasingly at risk in American society.

We treat this dataset as a record of production and climate outcomes only, not as a complete picture of the people who participate in the industry that these outcomes describe.

## Data quality caveats

We would like to provide transparency with respect to some key tradeoffs in our data collection:

- **Pre-2002 regional variety data is reconstructed, not directly reported.** From 1987–2001, the industry reports at the statewide level did not break variety out by region; those years were reconstructed by aggregating county-level figures up to reporting regions. North Willamette Valley acreage, for example, is undercounted by roughly 15–20% for years before 1999 because the source area tables for that era only list major counties. This undercount only affects the variety-level breakdown for that region; the “all varieties” total is accurate. We treat pre-2002 regional findings with corresponding caution throughout this report and do not present county-aggregated reconstructions as precise counts.
- **Regional and statewide panels begin in 1987, not 1981, despite full climate coverage back to 1981**: As described above, this is driven entirely by gaps in the wine production data source tables, not by any limitation in the climate data.
- **Price per ton (`PRI_01`) is only available at the region level from 2010 onward.** This is a genuine limitation of the source data and constrains how far back any price-based analysis can go. Additionally, because price is affected by many factors, of which climate is only one, our analysis centers mainly around production data rather than price data.
- **The wine census is self-reported and voluntary, not a full census.** We treat year-to-year volatility in the raw production series with more caution than the smoothed, multi-year trends.

None of these caveats change our core findings, which are drawn primarily from long-term trends and the more reliable 2002–2024 window and from the climate data which has no comparable gaps. PRISM provides complete daily coverage for all 23 AVAs across the full 1981–2024 period.

# Analysis

In [ ]:

library(arrow)



Attaching package: 'arrow'

The following object is masked from 'package:utils':

    timestamp

── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.2.1     ✔ readr     2.2.0
✔ forcats   1.0.1     ✔ stringr   1.6.0
✔ ggplot2   4.0.3     ✔ tibble    3.3.1
✔ lubridate 1.9.5     ✔ tidyr     1.3.2
✔ purrr     1.2.2     

── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ lubridate::duration() masks arrow::duration()
✖ dplyr::filter()       masks stats::filter()
✖ dplyr::lag()          masks stats::lag()
ℹ Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors


Attaching package: 'scales'

The following object is masked from 'package:purrr':

    discard

The following object is masked from 'package:readr':

    col_factor

Loading required package: zoo


Attaching package: 'zoo'

The following objects are masked from 'package:base':

    as.Date, as.Date.numeric

Loading required package: sandwich


Attaching package: 'strucchange'

The following object is masked from 'package:stringr':

    boundary

Loading required package: carData


Attaching package: 'car'

The following object is masked from 'package:dplyr':

    recode

The following object is masked from 'package:purrr':

    some

## Oregon’s growing reliance on Pinot Noir

In [ ]:
variety_share <- statewide %>%
  filter(!is.na(production_share_pct)
    , !is_zero_production
    , variety_label != "All"
    , yr >= 1987) %>%
  mutate(variety_group = if_else(variety_label %in% key_varieties, variety_label, "Other")) %>%
  group_by(yr, variety_group) %>%
  summarise(production_tons = sum(production_tons, na.rm = TRUE), .groups = "drop") %>%
  group_by(yr) %>%
  mutate(production_share_pct = production_tons / sum(production_tons, na.rm = TRUE) * 100) %>%
  ungroup()

labels_df <- variety_share %>%
  group_by(variety_group) %>%
  filter(yr == max(yr)) %>%
  ungroup()

variety_share %>%
  mutate(is_pinot = variety_group == "Pinot Noir") %>%
  ggplot(aes(yr, production_share_pct, color = variety_group, alpha = is_pinot, linewidth = is_pinot)) +
  geom_line(alpha = 0.2, linewidth = 0.4) +
  geom_smooth(method = "loess", se = FALSE, span = 0.4, aes(linewidth = is_pinot)) +
  geom_text_repel(
      data = labels_df, aes(x = yr, y = production_share_pct, label = variety_group, color = variety_group)
      , nudge_x = 1
      , hjust = 0
      , size = 3.2
      , segment.color = variety_colors_light
      , segment.size = 0.3
      , direction = "y"
      , show.legend = FALSE
      , inherit.aes = FALSE
    ) +
  scale_color_manual(values = variety_colors) +
  scale_linewidth_manual(values = c(`TRUE` = 1.4, `FALSE` = 0.8)) +
  scale_alpha_manual(values = c(`TRUE` = 1, `FALSE` = 0.7)) +
  scale_x_continuous(limits = c(1987, 2028), breaks = seq(1987, 2024, 5)) +
  labs(title = "Pinot Noir has come to define Oregon wine"
       , subtitle = "Share of statewide tonnage by variety, 1987–2024"
       , x = NULL, y = "Production share (%)") +
  theme_minimal() +
  theme(legend.position = "none"
        , plot.title = element_text(size = 16, face = "bold")
        , plot.subtitle = element_text(size = 10, color = "grey40"))


`geom_smooth()` using formula = 'y ~ x'

Figure 1 demonstrates Pinot Noir’s growing share of statewide tonnage, from roughly 15% in 1990 to around 30% by 2020. This trend has continued to build, while other anchor varieties like Chardonnay and Riesling have declined sharply over the same period. Pinot Gris is one variety that counteracts this trend, growing from a negligible share in 1990 to a stable ~8% by 2010.

However, the dramatic shift toward Pinot Noir has remained a dominant trend, suggesting that Oregon’s wine industry has increasingly concentrated on climate-sensitive Pinot varieties over exactly the same decades that have coincided with a pronounced shift in the region’s key climate indicators.

## Evidence of a warming climate

In [ ]:
sw_climate <- statewide %>%
  distinct(
    yr
    , gdd_apr_sep
    , gdd_apr_oct
    , frost_days_annual
    , frost_days_spring
    , heat_stress_days
    , summer_tmax_mean
    , oct_ppt_mm
    , annual_ppt_mm
    , vpd_max_summer
    , diurnal_range_ripening
    ) %>%
  arrange(yr) %>%
  mutate(
    gdd_rolling10 = zoo::rollmean(gdd_apr_sep, 10, fill = NA, align = "right")
    , gdd_anom = gdd_apr_sep - mean(gdd_apr_sep, na.rm = TRUE)
    , gdd_rolling10_anom = gdd_rolling10 - mean(gdd_apr_sep, na.rm = TRUE)
    )

# Structural-break test on the raw GDD series
gdd_ts <- ts(sw_climate$gdd_apr_sep, start = min(sw_climate$yr))
bp_fit <- breakpoints(gdd_ts ~ 1)
break_idx  <- bp_fit$breakpoints[1]
break_year <- sw_climate$yr[break_idx]

ggplot(sw_climate, aes(yr, gdd_anom)) +
  geom_col(aes(fill = gdd_anom > 0), width = 0.7) +
  geom_line(aes(y = gdd_rolling10_anom), color = "black", linewidth = 1) +
  geom_hline(yintercept = 0, color = "grey30", linewidth = 0.4) +
  scale_fill_manual(values = c(`TRUE` = "#8B1E3F", `FALSE` = "#2166ac"), guide = "none") +
  geom_vline(xintercept = break_year, linetype = "dashed", color = "grey40", linewidth = 0.6) +
  annotate("text", x = break_year, y = -150, label = paste0("Structural shift begins in ", break_year), hjust = -0.15, vjust = 1, size = 3, color = "grey40") +
  labs(title = "Oregon's growing season is getting warmer"
       , subtitle = "Growing Degree Days (GDD) anomaly from the 1987–2024 mean"
       , x = NULL, y = "GDD (Apr–Sep) anomaly") +
  theme_minimal() +
  theme(legend.position = "none"
      , plot.title = element_text(size = 16, face = "bold")
      , plot.subtitle = element_text(size = 10, color = "grey40"))


(`geom_line()`).

Growing degree days (GDDs) show a clear and sustained upward trend, with a level shift identified by a structural-break test around 2012.

In [ ]:
sw_climate %>%
  select(yr
         , `Frost risk days` = frost_days_spring
         , `GDD (Apr-Sep)` = gdd_apr_sep
         , `Peak summer temperature (°C)` = summer_tmax_mean
         , `Heat stress days` = heat_stress_days
         , `Diurnal ripening temperature range (°C)` = diurnal_range_ripening
         , `October precipitation (mm)` = oct_ppt_mm
         ) %>%
  pivot_longer(-yr, names_to = "indicator", values_to = "value") %>%
  mutate(indicator = factor(indicator, levels = c(
    "Frost risk days"
    , "GDD (Apr-Sep)"
    , "Peak summer temperature (°C)"
    , "Heat stress days"
    , "Diurnal ripening temperature range (°C)"
    , "October precipitation (mm)"
  ))) %>%
  ggplot(aes(yr, value)) +
  geom_line(color = "grey60", alpha = 0.7) +
  geom_smooth(method = "lm", se = TRUE, color = "#8B1E3F", fill = "#8B1E3F", alpha = 0.15) +
  facet_wrap(~indicator, scales = "free_y", ncol = 2) +
  labs(title = str_wrap("Diurnal ripening range is a notable exception among Oregon's viticultural indicators", width = 60)
       , subtitle = "Oregon viticultrual indicators by season; linear trend lines with 95% confidence intervals, 1987-2024"
       , x = NULL, y = NULL) +
  theme_minimal() +
  theme(legend.position = "none"
      , plot.title = element_text(size = 16, face = "bold")
      , plot.subtitle = element_text(size = 10, color = "grey40"))


`geom_smooth()` using formula = 'y ~ x'

Figure 3 summarizes six key viticultural climate indicators. Growing degree days, peak summer temperature, and heat stress days are all trending up with tight confidence intervals, demonstrating a warming signal is both real and steep. Spring frost days are essentially flat, a meaningful non-finding since it means the warming trend has not reduced the risk of a damaging frost event during spring bud break, even as summer heat accumulation climbs. October precipitation trends slightly upward but noisily, with no strong signal either way.

The sixth indicator—ripening-season diurnal temperature range—is the one metric that moves in the opposite direction from the rest, and it does so specifically because it isn’t a simple byproduct of an overall upward heat trend. This finding is most directly relevant to Oregon’s growing dependence on Pinot Noir, since Pinot Noir relies on a wide day-night temperature swing during ripening to retain acidity and develop aromatic complexity—a mechanism we discussed in [Background](#background).

## How the climate features relate

Several of these climate measures are closely related to aspects of the same underlying warming trend.

In [ ]:
# Aggregate to one row per year (mean across regions) before computing correlations
ry <- regional %>%
  distinct(
    region_label
    , yr
    , gdd_apr_sep
    , gdd_veraison_harvest
    , frost_days_spring
    , heat_stress_days
    , coldest_spring_night_c
    , oct_ppt_mm
    , sep_oct_ppt_mm
    , gs_ppt_mm
    , vpd_max_summer
    , diurnal_range_ripening
    )

climate_yearly <- ry %>%
  group_by(yr) %>%
  summarise(across(
    c(gdd_apr_sep, gdd_veraison_harvest, frost_days_spring, heat_stress_days
      , coldest_spring_night_c, oct_ppt_mm, sep_oct_ppt_mm, gs_ppt_mm
      , vpd_max_summer, diurnal_range_ripening)
    , mean, na.rm = TRUE
  ), .groups = "drop")


ℹ In argument: `across(...)`.
ℹ In group 1: `yr = 1987`.
Caused by warning:
! The `...` argument of `across()` is deprecated as of dplyr 1.1.0.
Supply arguments directly to `.fns` through an anonymous function instead.

  # Previously
  across(a:b, mean, na.rm = TRUE)

  # Now
  across(a:b, \(x) mean(x, na.rm = TRUE))

Figure 4 identifies three clusters of variables that appear to measure largely the same underlying phenomenon from different angles, rather than independent drivers:

1.  **Spring frost risk:** Frost risk days and coldest spring night are correlated at -0.91, which makes sense given one is a temperature reading and the other is a threshold-count derived from it.
2.  **Summer heat accumulation:** Heat stress days and summer VPD are both correlated above 0.6 with all the GDD variables.
3.  **Fall precipitation:** Precipitation measures move together above 0.4.

We also note that diurnal ripening temperature range is negatively correlated with nearly every heat-related feature, consistent with the pattern described in [Background](#background), where nights warm faster than days as the season heats up overall, compressing the day-night swing. Additionally, growing-season precipitation is negatively correlated with GDD Apr-Sep (-0.11), indicating that hotter years also tend to be drier years overall.

To resolve which variables to retain for modeling, we turn to variance inflation factors (VIF) to capture multi-variable overlap.

In [ ]:
# Setup
set.seed(42)
plot_vif <- function(vif_vals) {
  data.frame(Variable = names(vif_vals), VIF = round(unname(vif_vals), 2)) %>%
    mutate(Variable = dplyr::recode(Variable
      , "frost_days_spring" = "Frost risk days"
      , "coldest_spring_night_c" = "Coldest spring night"
      , "gdd_veraison_harvest" = "GDD Veraison-Harvest"
      , "gdd_apr_sep" = "GDD Apr-Sep"
      , "heat_stress_days" = "Heat stress days"
      , "vpd_max_summer" = "Summer VPD"
      , "diurnal_range_ripening" = "Diurnal temp range"
      , "gs_ppt_mm" = "Growing season precip"
      , "sep_oct_ppt_mm" = "Sep-Oct precip"
      , "oct_ppt_mm" = "Oct precip"
    )) %>%
    arrange(desc(VIF)) %>%
    knitr::kable(caption = "Variance Inflation Factors", row.names = FALSE)
}

ry_clean <- ry %>%
  filter(!region_label %in% c("Eastern Oregon", "Other Region", "Other Oregon"))

vif_df <- ry_clean %>%
  mutate(dummy_outcome = rnorm(n()))

# VIF - all variables
vif_model <- lm(dummy_outcome ~ gdd_apr_sep
  + frost_days_spring
  + coldest_spring_night_c
  + gdd_veraison_harvest
  + heat_stress_days
  + vpd_max_summer
  + diurnal_range_ripening
  + sep_oct_ppt_mm
  + oct_ppt_mm
  , data = vif_df
  )
vif_vals <- car::vif(vif_model)
plot_vif(vif_vals)


  Variable                  VIF
  ---------------------- ------
  Sep-Oct precip           8.11
  Oct precip               6.75
  Summer VPD               6.43
  Coldest spring night     4.38
  GDD Apr-Sep              4.13
  Frost risk days          3.82
  Heat stress days         2.90
  GDD Veraison-Harvest     2.34
  Diurnal temp range       2.27

  : Variance Inflation Factors


Sep-Oct precipitation and Oct precipitation come out highest, confirming we should not include both in the model. We elected to drop Sep-Oct precipitation, as we recognize Oct precipitation specifically as an important botrytis / harvest-timing risk.

Surprisingly, Summer VPD is *higher* than GDD Apr-Sep (6.43 vs. 4.13), even though the correlation matrix showed VPD’s strongest pairwise correlation was with GDD. This tells us Summer VPD is well-explained by some *combination* of GDD Apr-Sep, heat stress days, and ripening temp range together, rather than by a strong individual pairwise correlation with any one of them. We elect to drop GDD Apr-Sep and keep Summer VPD, as it is the more direct measure of vine water stress.

In [ ]:
# VIF - final features
vif_model_2 <- lm(dummy_outcome ~ gdd_veraison_harvest 
  + frost_days_spring 
  + coldest_spring_night_c
  + heat_stress_days 
  + vpd_max_summer 
  + diurnal_range_ripening
  + oct_ppt_mm 
  , data = vif_df
  )
vif_vals <- car::vif(vif_model_2)
plot_vif(vif_vals)


  Variable                  VIF
  ---------------------- ------
  Coldest spring night     4.18
  Frost risk days          3.81
  Summer VPD               3.71
  Heat stress days         2.66
  GDD Veraison-Harvest     2.12
  Diurnal temp range       1.31
  Oct precip               1.15

  : Variance Inflation Factors


All remaining VIFs fall under a conservative threshold of VIF\<5. Our retained predictor set going into the modeling stage is:

| Feature | Definition |
|------------------------------------|------------------------------------|
| Frost risk days | Count of days below freezing during the April–May bud-break window |
| Coldest spring night | Minimum recorded temperature during spring bud-break |
| GDD (Veraison–Harvest) | Accumulated heat above a 10°C (50°F) base during July 15–October 15 “veraison-to-harvest” window |
| Heat stress days | Count of days with a maximum temperature above 35°C (95°F) during the growing season |
| Summer VPD | A measure of atmospheric dryness / evaporative demand during July–August |
| Diurnal temperature range (ripening) | Average day-night temperature swing during August–September, the period when grapes develop their sugar, color, and acidity |
| Harvest precipitation (October) | Total precipitation in October |

Note that GDD Apr-Sep itself remains central to the *descriptive* climate trends in Figures 2–3 above; it is excluded only from the predictive feature set, where its information is substantially redundant with the other retained heat measures.

# Results

In [ ]:

library(arrow)
library(tidyverse)
library(ggplot2)
library(scales)

regional  <- read_parquet("data/analysis_panel_regional.parquet")
statewide <- read_parquet("data/analysis_panel_statewide.parquet")

regional <- regional %>%
  mutate(yield_derived = production_tons / acres_harvested
         , variety_label = str_trim(variety_label)
         , variety_label = if_else(variety_label == "Riesling/White Riesling", "Riesling", variety_label))

statewide <- statewide %>%
  mutate(yield_derived = production_tons / acres_harvested
         , variety_label = str_trim(variety_label)
         , variety_label = if_else(variety_label == "Riesling/White Riesling", "Riesling", variety_label))

key_varieties <- c("Pinot Noir", "Pinot Gris", "Chardonnay", "Riesling")

variety_colors <- c(
  "Pinot Noir" = "#8B1E3F"
  , "Pinot Gris" = "#4393c3"
  , "Chardonnay" = "#d6604d"
  , "Riesling" = "#d4a017"
  , "Other" = "#999999"
)

region_levels <- regional %>%
  filter(!region_label %in% c("Eastern Oregon", "Other Region", "Other Oregon")) %>%
  distinct(region_label) %>%
  arrange(region_label) %>%
  pull(region_label)

# region_colors <- setNames(
#   RColorBrewer::brewer.pal(max(3, length(region_levels)), "Set1")[seq_along(region_levels)]
#   , region_levels
# )

region_colors <- c(
  "North Willamette Valley" = "#8B1E3F",
  "South Willamette Valley" = "#4393c3",
  "Rogue Valley"            = "#d6604d",
  "Umpqua"                  = "#d4a017",
  "Columbia River at Large" = "#999999"
)

lighten_color <- function(hex, amount = 0.65) {
  rgb_val <- grDevices::col2rgb(hex) / 255
  new_rgb <- rgb_val + (1 - rgb_val) * amount
  grDevices::rgb(new_rgb[1], new_rgb[2], new_rgb[3])
}
variety_colors_light <- setNames(sapply(variety_colors, lighten_color), names(variety_colors))
region_colors_light <- setNames(sapply(region_colors, lighten_color), names(region_colors))


## Production and yield have, so far, kept climbing

In [ ]:
statewide_yield <- statewide %>%
  filter(variety_label == "All"
         , !is.na(production_tons), !is.na(acres_harvested)
         , acres_harvested > 100) %>%   # drops the 2009 row (acres_harvested = 25.63, production missing)
  mutate(weighted_yield = production_tons / acres_harvested) %>%
  select(yr, weighted_yield) %>%
  arrange(yr)

ggplot(statewide_yield, aes(x = yr, y = weighted_yield)) +
  geom_line(color = "gray70", linewidth = 0.5, alpha = 0.6) +
  geom_smooth(data = statewide_yield %>% filter(yr <= 2001), method = "loess", se = FALSE, color = "#8B1E3F", linewidth = 1.3) +
  geom_smooth(data = statewide_yield %>% filter(yr >= 2002), method = "loess", se = FALSE, color = "#8B1E3F", linewidth = 1.3) +
  annotate("segment", x = 2020, xend = 2020, y = 0, yend = max(statewide_yield$weighted_yield, na.rm = TRUE) * 0.65
           , linetype = "dashed", color = "gray40", linewidth = 0.5) +
  annotate("text", x = 2021, y = max(statewide_yield$weighted_yield, na.rm = TRUE) * 0.6
           , label = str_wrap("2020: wildfire smoke taint", width = 15), hjust = 0, size = 3.2, color = "gray30", lineheight = 1) +
  labs(title = "Statewide production-weighted yield, 1987–2024",
       subtitle = "Smoothed trend, fit separately before/after 2002 methodology shift"
       , x = NULL, y = "Yield (tons per acre)") +
  theme_minimal() +
  theme(legend.position = "none"
      , plot.title = element_text(size = 16, face = "bold")
      , plot.subtitle = element_text(size = 10, color = "grey40"))


`geom_smooth()` using formula = 'y ~ x'
`geom_smooth()` using formula = 'y ~ x'

Figure 5 plots statewide production-weighted yield (total tons produced divided by total acres harvested) for every year from 1987 to 2024. This isolates how productive Oregon’s vineyard acreage has become over time, independent of how much new acreage has been planted. The thin gray line traces the raw annual totals as reported while the bold red trend line summarizes the underlying pattern using a smoothed fit. Harvested acreage expanded by roughly the same order of magnitude as raw production did over this period, which means most of the industry’s growth has come from planting more acres rather than from getting more productive per acre. Yield itself has seen modest growth, averaging around ~2.5 tons per acre in 1990s and 2000s and settling closer to ~3 tons per acre in the most recent decade.

We note a clear interruption in 2020, corresponding to a season with severe wildfire smoke taint that resulted in a substantial share of unusable fruit among that year’s harvest. This represents a discrete event rather than a gradual climate-trend effect.

In [ ]:
weighted_yield_variety <- statewide %>%
  filter(!is.na(production_tons), !is.na(acres_harvested), acres_harvested > 0
         , production_tons > 0
         , yr >= 1988
         , variety_label %in% key_varieties) %>%
  group_by(yr, variety_group = variety_label) %>%
  summarise(total_prod = sum(production_tons, na.rm = TRUE)
            , total_acres = sum(acres_harvested, na.rm = TRUE)
            , weighted_yield = total_prod / total_acres
            , .groups = "drop") %>%
  mutate(raw_color = variety_colors_light[variety_group]
         , trend_color = variety_colors[variety_group])

ggplot(weighted_yield_variety, aes(yr, weighted_yield)) +
  geom_line(aes(color = raw_color), linewidth = 0.5) +
  geom_smooth(aes(color = trend_color), method = "loess", se = FALSE, linewidth = 1.1) +
  scale_color_identity() +
  facet_wrap(~ variety_group, scales = "fixed") +
  labs(title = "Yield by variety, statewide"
       , subtitle = 'Production-weighted yield (total tons ÷ total harvested acres)'
       , x = NULL, y = "Yield (tons per acre)") +
  theme_minimal() +
  theme(legend.position = "none"
      , plot.title = element_text(size = 16, face = "bold")
      , plot.subtitle = element_text(size = 10, color = "grey40")
      , strip.text = element_text(face = "bold")
      , axis.text = element_text(size = 8)
      , panel.spacing = unit(1, "lines"))


`geom_smooth()` using formula = 'y ~ x'

In [ ]:
# Split years into cool/mid/hot growing seasons by statewide GDD, then compare
# each variety's weighted yield across those tercile buckets. Restricted to
# 2002+ for the same data-reliability reason cited elsewhere for statewide
# variety-level figures.
yearly_gdd <- statewide %>%
  filter(yr >= 2002) %>%
  distinct(yr, gdd_apr_sep) %>%
  arrange(yr) %>%
  mutate(gdd_tercile = ntile(gdd_apr_sep, 3)
         , gdd_tercile = factor(gdd_tercile, levels = 1:3, labels = c("cool", "mid", "hot")))

variety_temp_response <- weighted_yield_variety %>%
  filter(yr >= 2002) %>%
  inner_join(yearly_gdd %>% select(yr, gdd_tercile), by = "yr") %>%
  group_by(variety_group, gdd_tercile) %>%
  summarise(mean_yield = mean(weighted_yield, na.rm = TRUE), .groups = "drop") %>%
  pivot_wider(names_from = gdd_tercile, values_from = mean_yield) %>%
  mutate(pct_change_hot_vs_cool = (hot - cool) / cool * 100)

get_pct_change <- function(variety) {
  val <- variety_temp_response %>%
    filter(variety_group == variety) %>%
    pull(pct_change_hot_vs_cool)
  round(val, 0)
}


Figure 6 breaks down the production-weighted yield for each of the four leading varietals: Chardonnay, Pinot Gris, Pinot Noir, and Riesling. Each panel repeats the same visual structure as Figure 5: a thin line for the raw annual value and a bold smoothed trend layered on top.

We see a dramatic dip in Pinot Noir yield in 2020 while Pinot Gris and Chardonnay barely register the disruption comparatively. This selective impact is consistent with wildfire smoke taint, which binds more readily to red wine grapes because of their extended skin-contact fermentation process than to white wine production; anecdotally, 2020 also drove some producers toward “white Pinot” as a workaround, itself a sign of the industry’s adaptive capacity.

All varieties rebound to new all-time highs within three years. Indeed, outside of that one disrupted year, yield for Pinot Noir and the other key varieties has held steady or trended upward statewide, and the same holds when the data is cut by region instead of variety in Figure 7 (below). The historical pattern demonstrates that, within the range of GDD and heat stress Oregon has experienced thus far, warmer conditions have generally corresponded with higher yields per acre across nearly every variety and region.

However, this pattern is not uniform across varieties. Pinot Gris demonstrated the largest yield gain of the four key varieties (roughly a 35% increase), with Pinot Noir close behind (44%) and Chardonnay more modest (34%). In contrast, Riesling shows essentially no relationship between yield and warmth of the growing season (-2%). This could be attributed to the fact that its production is concentrated largely in the cooler North Willamette Valley that rarely experiences the warmer end of the state’s climate range.

We also note that Pinot Gris’ rising share of statewide production (Figure 1) is attributable not only to an increasing share of planted acres, but also because this variety appears to yield higher volumes in the warmer seasons that are becoming increasingly common in Oregon.

In [ ]:
weighted_yield_region <- regional %>%
  filter(!is.na(production_tons), !is.na(acres_harvested), acres_harvested > 0
         , production_tons > 0
         , yr >= 1988
         , !region_label %in% c("Eastern Oregon", "Other Region", "Other Oregon")) %>%
  group_by(yr, region_group = region_label) %>%
  summarise(total_prod = sum(production_tons, na.rm = TRUE)
            , total_acres = sum(acres_harvested, na.rm = TRUE)
            , weighted_yield = total_prod / total_acres
            , .groups = "drop") %>%
  mutate(raw_color = region_colors_light[region_group]
         , trend_color = region_colors[region_group])

ggplot(weighted_yield_region, aes(yr, weighted_yield)) +
  geom_line(aes(color = raw_color), linewidth = 0.5) +
  geom_smooth(aes(color = trend_color), method = "loess", se = FALSE, linewidth = 1.1) +
  scale_color_identity() +
  facet_wrap(~ region_group, scales = "fixed", ncol = 2, nrow = 3) +
  labs(title = "Yield by region"
       , subtitle = 'Production-weighted yield (total tons ÷ total harvested acres)'
       , x = NULL, y = "Yield (tons per acre)") +
  theme_minimal() +
  theme(legend.position = "none"
      , plot.title = element_text(size = 16, face = "bold")
      , plot.subtitle = element_text(size = 10, color = "grey40")
      , strip.text = element_text(face = "bold")
      , axis.text = element_text(size = 8)
      , panel.spacing = unit(1, "lines"))


`geom_smooth()` using formula = 'y ~ x'

Figure 7 repeats the same weighted-yield calculation from Figure 6, this time faceted by Oregon’s five wine-growing regions rather than by variety. Yield across nearly every region has held steady or trended upward, reinforcing our observation that, within the climate range Oregon has experienced so far, warming has generally not come at the cost of yield and in many regions it has coincided with higher yields overall.

Umpqua is the one region that breaks this pattern. To examine this trend in more detail, Table 1 (below) lays out four growing-season climate metrics for each region, averaged over 2002–2024, with regions ordered from warmest to coolest.

In [ ]:
region_climate_summary <- regional %>%
  filter(!region_label %in% c("Eastern Oregon", "Other Region", "Other Oregon")
         , yr >= 2002) %>%
  distinct(region_label, yr, gdd_apr_sep, summer_tmax_mean, heat_stress_days, diurnal_range_ripening) %>%
  group_by(region_label) %>%
  summarise(
    `Mean GDD (Apr-Sep)` = round(mean(gdd_apr_sep, na.rm = TRUE))
    , `Mean summer high (°C)` = round(mean(summer_tmax_mean, na.rm = TRUE), 1)
    , `Mean heat stress days` = round(mean(heat_stress_days, na.rm = TRUE), 1)
    , `Mean ripening diurnal range (°C)` = round(mean(diurnal_range_ripening, na.rm = TRUE), 1)
    , .groups = "drop"
  ) %>%
  arrange(desc(`Mean GDD (Apr-Sep)`)) %>%
  rename(Region = region_label)

knitr::kable(region_climate_summary)


  --------------------------------------------------------------------------
  Region            Mean GDD   Mean summer     Mean heat       Mean ripening
                   (Apr-Sep)     high (°C)   stress days  diurnal range (°C)
  -------------- ----------- ------------- ------------- -------------------
  Columbia River        1363          30.8          11.8                16.5
  at Large                                               

  Rogue Valley          1301          30.6           9.9                17.5

  Umpqua                1212          28.9           5.0                16.4

  North                 1162          27.8           4.3                14.6
  Willamette                                             
  Valley                                                 

  South                 1154          28.6           5.4                16.3
  Willamette                                             
  Valley                                                 
  --------------------------------------------------------------------------

  : Table 1: Average growing-season climate by region, 2002--2024,
  ranked warmest to coolest


Columbia River at Large is a clear outlier at the warm end, and the two Willamette Valley sub-regions anchor the cool end, with Rogue Valley and Umpqua occupying the middle of the range. We note that Umpqua ranks third of five regions by growing degree days, meaningfully warmer than either Willamette Valley sub-region, yet it has both the lowest-weighted yield of any region over 2002–2024 and by far the flattest yield trend over that period. In contrast, Columbia River at Large shows a yield trajectory that is consistent with other regions, indicating that the yield trend is not tied to temperature alone. Umpqua’s yield story may be driven by other factors unrelated to the climate variables tracked by our dataset. Potential factors could include variety mix, vineyard scale, soil composition, or terrain complexity. We flag it here as a pattern worth further investigation rather than a settled finding.

Our overall observation is that warmer temperatures appear to have limited or somewhat positive impact on yield. However, warmer seasons can raise yield while simultaneously degrading the flavor and quality characteristics that define Oregon’s premium wines, particularly Pinot Noir.

## The quality concern hiding under a healthy yield trend

In [ ]:
sw_climate <- statewide %>%
  distinct(yr, diurnal_range_ripening) %>%
  arrange(yr)

ggplot(sw_climate, aes(yr, diurnal_range_ripening)) +
  annotate("rect", xmin = -Inf, xmax = Inf, ymin = 15, ymax = 20
           , fill = "#4393c3", alpha = 0.08) +
  annotate("text", x = min(sw_climate$yr, na.rm = TRUE), y = 20.3
           , label = "Ideal range for Pinot Noir ripening (15-20°C)"
           , hjust = 0, size = 3, color = "#4393c3") +
  geom_line(color = "grey60", alpha = 0.7, linewidth = 0.6) +
  geom_smooth(method = "lm", se = TRUE, color = "#8B1E3F", fill = "#8B1E3F", alpha = 0.15, linewidth = 1.1) +
  labs(title = "Ripening-season diurnal temperature range is shrinking"
       , subtitle = "Day/night temperature swing during ripening (Aug-Sep), statewide average, 1981–2024"
       , x = NULL, y = "Diurnal temperature range (°C)") +
  theme_minimal() +
  theme(legend.position = "none"
      , plot.title = element_text(size = 16, face = "bold")
      , plot.subtitle = element_text(size = 10, color = "grey40"))


`geom_smooth()` using formula = 'y ~ x'

Figure 8 demonstrates a diurnal range that has narrowed from roughly 17°C in the 1980s to roughly 15°C today, indicating that nights are warming faster than days. This is consistent with a pattern documented globally in the climate literature and generally attributed to changes in cloud cover trapping heat overnight ([Dahl 2022](#ref-ucs2022nights); [<span class="nocase">Liu et al.</span> 2024](#ref-liu2024dtr)). This day-night temperature swing is a key mechanism behind Oregon’s suitability for growing Pinot Noir: warm days build sugar, while cool nights slow the grape’s respiration long enough to retain complex acidity, color, and aromatic qualities in the grape ([Seth D. Cohen et al. 2012](#ref-ajev2012dtr)). Controlled experiments on Pinot Noir, specifically, have found that compressing this range alters organic acid metabolism and can suppress color accumulation, and a companion study found Pinot Noir was one of the varieties most sensitive to elevated diurnal temperature among the major red varieties tested ([<span class="nocase">Bennett et al.</span> 2025](#ref-frontiers2025bunchheating); [Rosas et al. 2022](#ref-pmc2022highTemp)).

This narrowing diurnal range is a potential risk to wine *quality* that has not surfaced in the production metrics alone. Diurnal range operates on wine quality through a physiological process that is more nuanced than heat-accumulation metrics in isolation, and models built on yield alone will miss this leading indicator entirely. Future research could evaluate this risk through proxy indicators of wine quality such as price premiums or critic-score text mining.

## Predicting yield across the growing season

Next, we aim to create a predictive model built on two key observations from our analysis thus far: (1) yield has been resilient across the historical climate range and (2) the climate signal accumulates in stages over the course of the growing season. We argue that a model built only on a single end-of-season snapshot is less useful to a wine grower who experiences these climate conditions sequentially throughout the annual growth cycle.

Insights derived from our predictive model are intended to help wine growers understand not only what climate interactions have the ability to impact yield, but when those impacts would be most visible in the growing season, and for which varieties they matter most. We modeled yield_derived (production tons per harvested acre) using a Support Vector Machine (SVM) with a radial basis function (RBF) kernel, tuned via Cross-Validation (CV). An SVM was selected because the usable sample, roughly 1,000 region-by-variety-year observations, is small enough that a kernel method with relatively few tunable parameters is less prone to overfitting, while the RBF kernel still captures the non-linear climate-yield relationships suggested by our correlation analysis. Our small observation size contained class imbalances in varietal observations, particularly in Merlot and Pinot Blanc, where Pinot Blanc’s imbalance ratio against Pinot Noir was 3.7x, that were balanced in the training data through weighting samples by inverse frequency.

## Adjusting for class imbalances

- Weight = inverse of how many rows that variety-by-region combination has, normalized so weights average to 1
- Down-weights the influence of well-represented groups (Pinot Noir, North Willamette Valley) relative to thin ones (Pinot Blanc, Merlot)

Table 2 shows the resulting sample weight for each region-by-variety combination, ranked from most to least up-weighted. Groups with the fewest rows receive the largest weights, so the model treats each combination as roughly equally informative during training rather than letting the best-sampled groups (Pinot Noir, Chardonnay) dominate simply because they have more rows.

In [ ]:
sample_weights <- read_csv("data/ml_results/sample_weights.csv", show_col_types = FALSE) %>%
  arrange(desc(sample_weight))

sample_weights_trimmed <- bind_rows(
  head(sample_weights, 10) %>% mutate(group = "Most up-weighted (thinnest groups)"),
  tail(sample_weights, 5) %>% mutate(group = "Least up-weighted (best-sampled groups)")
)

knitr::kable(
  sample_weights_trimmed,
  col.names = c("Region", "Variety", "N rows", "Sample weight", "")
)


  ----------------------------------------------------------------------------
  Region          Variety          N rows    Sample 
                                             weight 
  --------------- ---------------- ------ --------- --------------------------
  Columbia River  Pinot Gris           20      1.43 Most up-weighted (thinnest
  at Large                                          groups)

  South           Merlot               20      1.43 Most up-weighted (thinnest
  Willamette                                        groups)
  Valley                                            

  Rogue Valley    Pinot Blanc          21      1.36 Most up-weighted (thinnest
                                                    groups)

  North           Pinot Blanc          21      1.36 Most up-weighted (thinnest
  Willamette                                        groups)
  Valley                                            

  Umpqua          Pinot Gris           23      1.24 Most up-weighted (thinnest
                                                    groups)

  Columbia River  Gewurztraminer       24      1.19 Most up-weighted (thinnest
  at Large                                          groups)

  Columbia River  Merlot               25      1.14 Most up-weighted (thinnest
  at Large                                          groups)

  Umpqua          Merlot               25      1.14 Most up-weighted (thinnest
                                                    groups)

  Rogue Valley    Merlot               25      1.14 Most up-weighted (thinnest
                                                    groups)

  Umpqua          Cabernet             27      1.06 Most up-weighted (thinnest
                  Savignon                          groups)

  North           Pinot Noir           31      0.92 Least up-weighted
  Willamette                                        (best-sampled groups)
  Valley                                            

  North           Gewurztraminer       31      0.92 Least up-weighted
  Willamette                                        (best-sampled groups)
  Valley                                            

  North           Chardonnay           31      0.92 Least up-weighted
  Willamette                                        (best-sampled groups)
  Valley                                            

  Columbia River  Riesling/White       31      0.92 Least up-weighted
  at Large        Riesling                          (best-sampled groups)

  Umpqua          Riesling/White       31      0.92 Least up-weighted
                  Riesling                          (best-sampled groups)
  ----------------------------------------------------------------------------

  : Table 2: Sample weight by region and variety combination (most and
  least up-weighted)


## Fitting three models to represent a sequential growing season

To capture the compounding effect of time and climate information, we fit three separate models, each representing what a grower would actually know at that point in the season: a spring model (frost risk only), a summer model (adding growing degree days, heat stress, and vapor pressure deficit), and a harvest model (adding veraison-to-harvest GDD, October precipitation, and ripening diurnal range).

All three stages were evaluated on the same time-blocked pre-2020 train/test partition rather than a simple random split. Oregon’s yield has trended upward over the study period, and its climate is itself autocorrelated from one year to the next, so a random split risks placing a test year directly adjacent to a training year from the same short-term trend, letting the model interpolate between nearby years rather than genuinely predicting an unseen one. To guard against this, we split pre-2020 data into contiguous three-year blocks, assigned roughly 10% of rows to test blocks spread across the full date range, and removed a one-year buffer from training on each side of the test block, so that no training year sits immediately adjacent to a test year. All three growing-season stages were tuned and evaluated on this same partition, so that any difference in performance across stages reflects the additional climate information available at that stage, not a different sample of years.

Post-2020 data was held out entirely from training and evaluated separately, as a case study rather than a fourth fold. We treat 2020 onward this way throughout the report because the wildfire smoke taint that disrupted that year’s harvest was a discrete event rather than part of the underlying gradual climate trend our models are built to capture; folding it into training on equal footing with other years would risk attributing a smoke-driven production loss to ordinary climate variability.

For a baseline metric against the SVM’s performance, we ran a weighted least squares linear model along the same sample splits. If the linear model found a relationship the SVM did not, that would point to the SVM being poorly suited to this data, rather than the underlying relationship being genuinely weak. Conversely, if the SVM detected a relationship the linear model could not, that would suggest a non-linear climate-yield relationship the RBF kernel is able to capture but a linear model cannot.

## Climate becomes predictive mid-season, then declines

In [ ]:
stage_metrics <- read_csv("data/ml_results/stage_model_metrics.csv", show_col_types = FALSE) %>%
  filter(model == "svm", group == "pooled") %>%
  mutate(stage = dplyr::recode(stage,
    stage1_spring  = "Stage 1: Spring",
    stage2_summer  = "Stage 2: Summer",
    stage3_harvest = "Stage 3: Harvest"
  )) %>%
  select(Stage = stage, RMSE = rmse, R2 = r2) %>%
  mutate(across(where(is.numeric), ~round(.x, 3)))

knitr::kable(stage_metrics, col.names = c("Stage", "RMSE", "R\u00b2"))


  Stage                 RMSE      R²
  ------------------ ------- -------
  Stage 1: Spring      1.139   0.012
  Stage 2: Summer      1.105   0.069
  Stage 3: Harvest     1.123   0.040

  : Table 3: Held-out test performance by growing-season stage (SVM)


In [ ]:
stage_predictions <- read_parquet("data/ml_results/stage_predictions.parquet") %>%
  mutate(residual = yield_derived - pred_svm
         , stage = dplyr::recode(stage,
             stage1_spring  = "Stage 1: Spring",
             stage2_summer  = "Stage 2: Summer",
             stage3_harvest = "Stage 3: Harvest"
           ))

ggplot(stage_predictions, aes(x = stage, y = residual)) +
  geom_hline(yintercept = 0, linetype = "dashed", color = "grey40") +
  geom_boxplot(fill = "#4393c3", alpha = 0.3, width = 0.5) +
  labs(title = "Residual spread widens at the harvest stage",
       subtitle = "Positive = model underestimated actual yield; negative = model overestimated"
       , x = NULL, y = "Residual (actual \u2212 predicted yield, tons/acre)") +
  theme_minimal()


Table 3 and Figure 9 show held-out test performance for all three growing-season stages. The relationship between climate and yield is modest throughout, an R² range of roughly 0.01 to 0.07. The staged models show that climate information is not predictive in the spring, becomes predictive in the summer, and then declines in predictive power once harvest-stage features are added. This pattern aligns with our correlation analysis, which found that growing degree days and heat stress were the most strongly correlated climate features with yield, while veraison-to-harvest GDD, October precipitation, and ripening diurnal range were not strongly correlated with yield.

Figure 9 shows a tighter spread of residuals for the summer model than for the harvest model, despite the harvest model having strictly more information available to it. Suggesting that the harvest-stage features are not adding predictive value and may even be introducing noise into the model. Because of this, we use the summer model, not the full-season harvest model, as our final model for the post-2020 case study below.

In [ ]:

reds <- c("Pinot Noir", "Cabernet Savignon", "Merlot")
whites <- c("Chardonnay", "Pinot Gris", "Riesling/White Riesling", "Gewurztraminer", "Pinot Blanc")

stage_predictions <- stage_predictions %>%
  mutate(
    wine_type = case_when(
      variety_label %in% reds ~ "Red",
      variety_label %in% whites ~ "White",
      TRUE ~ "Other"
    ),
    # whites first (bottom), reds last (top) after coord_flip
    variety_label = factor(variety_label, levels = c(whites, reds))
  )

p_residuals_by_variety <- ggplot(stage_predictions, aes(x = variety_label, y = residual, fill = wine_type)) +
  geom_hline(yintercept = 0, linetype = "dashed", color = "black") +
  geom_boxplot(alpha = 0.3) +
  coord_flip() +
  facet_wrap(~stage) +
  scale_fill_manual(values = c("Red" = "#881e3f", "White" = "#4393c3", "Other" = "grey60")) +
  labs(title = "Which varieties does the model explain well,\nwhere does it struggle?",
       subtitle = "Positive = model underestimated actual yield;\nnegative = model overestimated",
       x = NULL, y = "Residual (actual \u2212 predicted yield, tons/acre)",
       fill = "Wine type") +
  theme_minimal() +
  theme(legend.position = "bottom",
        strip.placement = "outside",
        strip.text = element_text(face = "bold", size = 9),
        plot.title = element_text(size = 16, face = "bold"),
        plot.subtitle = element_text(size = 13, color = "grey40"),
        axis.text = element_text(size = 10, color = "black"),
        axis.title = element_text(size = 11, color = "black"))

p_residuals_by_variety


Figure 10 breaks down the same residuals by variety instead of by stage. As shown, the model is best-behaved for Oregon’s most heavily represented varieties: Pinot Noir and Chardonnay in particular show residuals centered close to zero across all three stages, while the state’s thinner varieties show wider, more biased spreads that do not meaningfully improve as later-stage features are added. This behavior becomes more explicit once we look at which features are contributing the most predictive power in the model. Ultimately, when climate information alone cannot fully explain a variety’s yield, the model falls back on variety identity itself as its best available predictor.

## Model reliance on variety identity vs. climate features

In [ ]:
var_labels <- c(
  "gdd_apr_sep"            = "GDD (Apr–Sep)",
  "gdd_apr_oct"            = "GDD (Apr–Oct)",
  "gdd_veraison_harvest"   = "GDD (Veraison–Harvest)",
  "frost_days_spring"      = "Spring Frost Days",
  "frost_days_annual"      = "Annual Frost Days",
  "heat_stress_days"       = "Heat Stress Days (>35°C)",
  "coldest_spring_night_c" = "Coldest Spring Night (°C)",
  "winter_tmin_jan_feb"    = "Winter Tmin (Jan–Feb)",
  "summer_tmax_mean"       = "Summer Tmax Mean (°C)",
  "diurnal_range_ripening" = "Diurnal Range (Ripening)",
  "gs_ppt_mm"              = "Growing Season Precip (mm)",
  "oct_ppt_mm"             = "October Precip (mm)",
  "sep_oct_ppt_mm"         = "Sep–Oct Precip (mm)",
  "spring_ppt_mm"          = "Spring Precip (mm)",
  "annual_ppt_mm"          = "Annual Precip (mm)",
  "vpd_max_summer"         = "Max Summer VPD",
  "vpd_max_ripening"       = "Max Ripening VPD",
  "variety_label"          = "Variety Label"
)

stage_importance <- read_parquet("data/ml_results/stage_importance.parquet") %>%
  mutate(stage = dplyr::recode(stage,
    stage1_spring  = "Stage 1: Spring",
    stage2_summer  = "Stage 2: Summer",
    stage3_harvest = "Stage 3: Harvest"
  ),
  Variable_clean = dplyr::recode(Variable, !!!var_labels)
  )

stage_importance %>%
  group_by(stage) %>%
  slice_max(Importance, n = 5) %>%
  ungroup() %>%
  mutate(Variable_key = paste(stage, Variable_clean, sep = "___")) %>%
  ggplot(aes(x = reorder(Variable_key, Importance), y = Importance)) +
  geom_col(fill= "#8B1E3F")+
  coord_flip() +
  scale_x_discrete(labels = function(x) sub("^.*___", "", x)) +
  facet_wrap(~stage, scales = "free_y", ncol = 1) +
  labs(title = "Which features actually move the model's predictions",
       x = NULL, y = "Importance (RMSE increase when permuted)") +
  theme_minimal()+
    theme(
    panel.spacing.y = unit(1.2, "lines"),
    strip.text = element_text(face = "bold", size = 11, margin = margin(b = 6)),
    axis.text.y = element_text(size = 9, color = "black"),
    axis.text.x = element_text(size = 8.5, color = "black"),
    axis.title.x = element_text(size = 10, margin = margin(t = 10)),
    plot.title = element_text(face = "bold", size = 13, margin = margin(b = 10))
  )


Figure 11 shows why the residual patterns in Figure 10 look the way they do. In the spring model, variety identity dominates by a wide margin, consistent with the only available features at that point, frost risk and coldest spring night, showing no meaningful correlation with yield in our statistical analysis. With no useful climate signal to draw on, the model falls back on variety as its primary basis for prediction.

Variety identity remains the most important feature in the summer model as well, but growing degree days now emerges as a substantial contributor alongside it. This lines up directly with Table 3: the summer model’s jump in R² is attributable specifically to GDD becoming informative once it enters the feature set, not to any change in how much the model relies on variety. Figures 10 and 11 reveal the influence variety has on yield, while also highlighting GDD as an important climate predictor, however as climate factors accumulate the model does not display increased accuracy or a reduction in the model’s reliance on variety.

## Does the post-2020 period leave a more visible mark on some varieties?

Treating 2020 as a discrete disruption rather than part of the underlying climate trend, we can evaluate how well the summer model explains each variety’s yield from 2020 through 2024. We refit the summer model, the best-performing stage, on all pre-2020 data and evaluated it against each variety’s actual yield from 2020 through 2024. A wide or biased residual here does not mean climate hit that variety harder; the summer model has no wildfire or smoke-taint feature at all, so a large residual only tells us climate and variety together fail to explain that variety’s post-2020 yield, for reasons the model cannot see.

In [ ]:
post2020_summary <- read_parquet("data/ml_results/post2020_case_study.parquet") %>%
  group_by(variety_label) %>%
  summarise(mean_resid = round(mean(residual, na.rm = TRUE), 2)
            , sd_resid = round(sd(residual, na.rm = TRUE), 2)
            , n = n()
            , .groups = "drop") %>%
  arrange(desc(sd_resid))

knitr::kable(post2020_summary,
             col.names = c("Variety", "Mean residual", "SD of residual", "N (2020-2024 rows)"))


  -----------------------------------------------------------------------
  Variety                 Mean residual SD of residual N (2020-2024 rows)
  ----------------------- ------------- -------------- ------------------
  Pinot Blanc                     -0.28           1.39                 10

  Riesling/White Riesling         -0.23           1.24                 25

  Gewurztraminer                  -0.10           1.02                 20

  Chardonnay                       0.09           0.97                 25

  Cabernet Savignon                0.17           0.95                 19

  Pinot Gris                       0.76           0.89                 17

  Pinot Noir                       0.45           0.74                 25

  Merlot                          -0.07           0.60                 15
  -----------------------------------------------------------------------

  : Table 4: Post-2020 residuals by variety (summer model), 2020-2024


In [ ]:
ggplot(post2020_summary, aes(x = reorder(variety_label, sd_resid), y = mean_resid)) +
  geom_hline(yintercept = 0, linetype = "dashed", color = "grey40") +
  geom_pointrange(aes(ymin = mean_resid - sd_resid, ymax = mean_resid + sd_resid),
                   color = "#8B1E3F", size = 0.6) +
  coord_flip() +
  labs(title = "How well does the summer climate model explain each variety, 2020-2024?",
       subtitle = "Mean residual \u00b1 1 SD; positive = actual yield exceeded prediction"
       , x = NULL, y = "Residual (actual \u2212 predicted yield, tons/acre)") +
  theme_minimal()


Table 4 and Figure 12 show that post-2020 unpredictability is not evenly spread across varieties, and it does not fall where our earlier smoke-taint finding might suggest. Pinot Blanc and Riesling show the widest residual spread and the most negative mean, meaning their yeild is consistently lower than the model predicts, while Chardonnay and Pinot Gris show the tightest residual spread and a mean near zero. Pinot Blanc’s result, drawn from only 10 rows, is the thinnest sample in this case study and should be read with particular caution.

Pinot Noir, the variety our production-weighted yield analysis showed took the sharpest single-year hit from 2020 wildfire smoke, tells a different story once the window widens to include the recovery years. Its residual spread is the second-tightest of any variety, and its mean residual is solidly positive, meaning actual yield from 2020 through 2024 ran above what climate alone would have predicted. Our framing of 2020 as a discrete, one-year disruption holds up once the window widens, not just for that single year: across the full post-2020 period, Pinot Noir’s climate-yield relationship looks stable, and the industry’s documented rebound (Figure 6) shows up here as a model that continues to underestimate, not overestimate, the variety’s recovery.

Since our summer model was never built to detect wildfire smoke specifically, we read Pinot Blanc and Riesling’s post-2020 unpredictability as a starting point for further investigation rather than a settled explanation. Whether it reflects a genuine variety-specific climate response, the compounding effect of multiple fire years our features do not capture, or simply the smaller and noisier samples these two varieties carry throughout this analysis, is a question this dataset alone cannot resolve.

# Conclusion

In [ ]:
library(arrow)
library(tidyverse)
library(ggplot2)

statewide <- read_parquet("data/analysis_panel_statewide.parquet") %>%
  mutate(variety_label = str_trim(variety_label))
regional  <- read_parquet("data/analysis_panel_regional.parquet")
regional <- regional %>%
  mutate(yield_derived = production_tons / acres_harvested
         , variety_label = str_trim(variety_label)
         , variety_label = if_else(variety_label == "Riesling/White Riesling", "Riesling", variety_label))


Our original research question asked how climate has affected wine production in Oregon’s growing regions, and how that relationship has evolved since 1987. Our statistical and machine learning findings point to an answer that is more nuanced than either “climate change is hurting Oregon wine” or “climate change hasn’t mattered yet.” Both are somewhat accurate, depending on which outcome and which variety is the focus.

The climate trend itself is unambiguous. Growing degree days, peak summer temperature, and heat stress days have all risen over the past four decades, while the ripening-season diurnal temperature range, the day-night swing that gave Oregon’s Willamette Valley its Burgundy-like suitability for Pinot Noir in the first place, has quietly narrowed. Yield, however, has not shown the damage that trend alone might suggest. Statewide production-weighted yield has climbed for decades, our correlation analysis found warmer conditions generally corresponding to higher yield rather than lower, and our staged SVM model could explain at most 6.9% of yield’s variance from climate and variety alone, at its single best-performing stage. That is independent, held-out confirmation that climate is not yet the dominant force behind Oregon’s yield outcomes.

In [ ]:

library(patchwork)


`geom_smooth()` using formula = 'y ~ x'
`geom_smooth()` using formula = 'y ~ x'
`geom_smooth()` using formula = 'y ~ x'

`geom_smooth()` using formula = 'y ~ x'
`geom_smooth()` using formula = 'y ~ x'
`geom_smooth()` using formula = 'y ~ x'

Figure 13 highlights the quality concern we identified earlier. The metric the industry measures and reports every year, yield, has moved in one direction. The metric that actually explains why Oregon built its reputation on Pinot Noir in the first place has moved in the other, causing a tightening of the diurnal temperature range. This is a quality concern that has yet to appear anywhere in Oregon’s production statistics. Our machine learning results reinforce just how real that separation is: ripening diurnal range did not rank among the top five most important features for predicting yield at any stage of the growing season, including the harvest-stage model where it was directly available to the model as a feature. A grower, or a researcher, looking at yield alone has no reason to suspect anything has changed.

That resilience is also not evenly distributed across varieties, our machine learning results showed that when climate information was thin the model fell back almost entirely on which variety it was predicting rather than on climate itself. This is direct evidence that variety identity, not climate, is the first-order determinant of yield outcomes at this stage of Oregon’s warming. Once summer climate features entered, growing degree days emerged as a significant second contributor alongside variety, but variety identity remained the single most important feature at every stage we tested, spring, summer, and harvest alike. And the residual diagnostics showed that the varieties the model struggles to characterize, Cabernet Sauvignon, Merlot, and Pinot Blanc in particular, are disproportionately the state’s minor plantings. We have the least reliable climate-sensitivity evidence for exactly the varieties with the smallest foothold to begin with, and the thinnest historical record to learn from.

Our post-2020 case study aligns with our earlier findings while also complicating them. If weather were leaving a uniform mark across varieties, we would expect the variety our earlier analysis flagged as hit hardest by 2020 wildfire smoke, Pinot Noir, to also be the hardest for a climate model to explain across the recovery years that followed. It is not. Once the window widens to 2020 through 2024, Pinot Noir’s yield is among the best-explained of any variety, and consistently better than a climate-only model would predict, showing a genuine recovery rather than a lingering instability. The varieties our summer model struggles with most across this same period, Pinot Blanc and Riesling, are not the ones the smoke-taint story would have predicted, and Pinot Blanc’s result rests on a sample too thin to weigh heavily on its own. Weather does appear to leave a more visible mark on some varieties than others, but not necessarily the ones a single dramatic year would lead us to expect, and not always for reasons a climate-and-variety model alone can identify.

This same climate shift also coincided with a real industry response: a consolidation around Pinot Noir, with Chardonnay and Riesling’s statewide shares declining over the same period. Pinot Noir shows a moderate positive yield relationship with heat accumulation in our data, comparable to Pinot Gris and stronger than several other varieties, so nothing in our yield findings suggests this consolidation has come at a yield cost. But the diurnal-range finding is the one piece of evidence in this entire report that has not yet shown up in a production number, a leading indicator for a wine-quality risk that a yield-only model, ours included, is structurally unable to see.

## Limitations

Several limitations shape how far these findings should be read. Our usable machine learning sample, roughly 1,000 region-by-variety-year observations after filtering, is small enough that individual thin varieties carry real uncertainty even after correcting for class imbalance through inverse-frequency sample weighting; we verified this weighting scheme against the alternative of synthetic oversampling (SMOTE) and found it produced more consistent train and test distributions for this dataset. An R² below 0.07 at every stage means our model should be read as evidence about where and for which varieties climate signal is present in the yield record, not as a production-forecasting tool. Umpqua’s yield pattern, flat and among the lowest of any region despite being the third-warmest by growing degree days, remains unexplained by the climate variables in our dataset and is flagged rather than resolved. Because yield was our only modeled outcome, these results say nothing directly about price or the qualitative characteristics that matter most to Oregon’s fine wine reputation, a limitation that reflects our finding that harvest-window features added no predictive value for yield specifically.

## Future work

The honest answer to our original research question is that Oregon’s climate-wine relationship has evolved in a way that is currently invisible in the production data, and will most plausibly first appear in the fruit quality of the state’s most heat-sensitive, most heavily planted variety. That is precisely the risk this project’s founding motivation, building a public, climate-linked Oregon wine dataset where none currently exists, was designed to help detect. It is also where we would point future work: extending this model to a genuine quality proxy, such as price premium or critic-score data, rather than yield alone, disaggregating the thin varieties our current sample cannot yet characterize with confidence as more years of data accumulate, and extending the comparison to California and Washington that our original proposal envisioned but our first-phase data constraints did not allow.

# References

<span class="nocase">Bennett, Jeffrey S. et al.</span> 2025. “Effects of Bunch Heating on Pinot Noir Grape Ripening: Changes in Berry Anthocyanin and Malic Acid Concentrations and Their Regulation.” *Frontiers in Plant Physiology*. <https://www.frontiersin.org/journals/plant-physiology/articles/10.3389/fphgy.2025.1701827/abstract>.

Cohen, Sarah D., Julie M. Tarara, Gregory A. Gambetta, Mark A. Matthews, and James A. Kennedy. 2012. “Impact of Diurnal Temperature Variation on Grape Berry Development, Proanthocyanidin Accumulation, and the Expression of Flavonoid Pathway Genes.” *Journal of Experimental Botany* 63 (7): 2655–65. <https://doi.org/10.1093/jxb/err449>.

Cohen, Seth D., Julie M. Tarara, and James A. Kennedy. 2012. “Diurnal Temperature Range Compression Hastens Berry Development and Modifies Flavonoid Partitioning in Grapes.” *American Journal of Enology and Viticulture* 63 (1): 112–26. <https://www.ajevonline.org/content/63/1/112>.

Dahl, Kristina. 2022. *With Climate Change, Nights Are Warming Faster Than Days. Why?* Union of Concerned Scientists Blog. <https://blog.ucs.org/kristy-dahl/with-climate-change-nights-are-warming-faster-than-days-why/>.

Delelee, L. M., A. J. Woodill, and P. A. Skinkis. 2025. “Climate Change Projections Indicate Shifts in Phenology for Willamette Valley Pinot Noir.” *American Journal of Enology and Viticulture*.

Jones, Gregory V. 2005. “Climate Change in the Western United States Grape Growing Regions.” *Acta Horticulturae*.

Jones, Gregory V., and Edward Hellman. 2003. “Site Assessment.” In *Oregon Viticulture*, 5th ed., edited by Edward Hellman. Oregon State University Press.

Kliewer, W. M., and R. E. Torres. 1972. “Effect of Controlled Day and Night Temperatures on Grape Coloration.” *American Journal of Enology and Viticulture* 23 (2): 71–77. <https://www.ajevonline.org/content/23/2/71>.

<span class="nocase">Liu, X. et al.</span> 2024. “Increased Asymmetric Warming Rates Between Daytime and Nighttime Temperatures over Global Land During Recent Decades.” *Geophysical Research Letters*. <https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2024GL112832>.

Oregon Wine Board. 2026. *Oregon Vineyard and Winery Census*. <https://industry.oregonwine.org/research/oregon-vineyard-and-winery-report/>.

Oregon Wine Board Trade Resources. 2025. *Willamette Valley*. <https://trade.oregonwine.org/resources/willamette-valley/>.

PRISM Climate Group, Oregon State University. 2026. *PRISM Climate Group*. <https://www.prism.oregonstate.edu/>.

Rosas, Inés de, Leonor Deis, Yésica Baldo, Juan B. Cavagnaro, and Pablo F. Cavagnaro. 2022. “High Temperature Alters Anthocyanin Concentration and Composition in Grape Berries of Malbec, Merlot, and Pinot Noir in a Cultivar-Dependent Manner.” *Plants*. <https://pmc.ncbi.nlm.nih.gov/articles/PMC9003205/>.

Skahill, B., and B. Berenguer. 2025. *Temperature-Based Climate Projections of Pinot Noir Suitability in the Willamette Valley AVA*. Oregon Wine Symposium.

SommSelect. 2026. *Willamette Valley Wine Guide – Pinot Noir & Top AVAs*. <https://sommselect.com/blogs/tasting-notes/willamette-valley-wine-guide-history-regions-and-top-wines>.

<span class="nocase">Van Leeuwen, C. et al.</span> 2024. “Climate Change Impacts and Adaptations of Wine Production.” *Nature Reviews Earth & Environment*.

Wine Data Researcher. 2022. *The 2022 US Census of Agriculture: Insights for Wine Organizations and Associations*. <https://winedataresearcher.com/the-2022-us-census-of-agriculture-insights-for-wine-organizations-and-associations/>.

WineAmerica. 2025. *Economic Impact Study 2025: Oregon Wine Industry*. <https://wineamerica.org/economic-impact-study-2025/oregon-wine-industry-2025/>.